tests/test_json.py

```diff
@@ -137,12 +142,16 @@ def test_rrule_timezone(self):
 class RedBeatJSONDecoderTestCase(JSONTestCase):
 
     def test_datetime(self):
-        d = self.datetime()
+        for zone in (None, 'UTC', 'Asia/Shanghai'):
```

Comment:
Rather than loop over these within the tests, should it be split into two tests,  test_datetime_default, test_datetime_provided or similar?   I find it's easier to debug test failures if the test name is explicit about the behaviour its testing.  The common code could be factored out into an _assert_something() method.

Reply:
ok